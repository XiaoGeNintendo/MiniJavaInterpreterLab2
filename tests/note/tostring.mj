int main(){
    char[] cArr = {'1', '2' ,'3'};
    string str = to_string(cArr); // 123
    println(cArr);
    println(str);
    char[] cArr1 = null;
    string str1 = to_string(cArr1); // null pointer error
    return 0;
}
