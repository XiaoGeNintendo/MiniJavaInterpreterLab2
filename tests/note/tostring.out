[1, 2, 3]
123
Process exits with the code 34.